src/lib/containers/widgets/query-filter/EnumFacetFilter.tsx

```diff
@@ -8,7 +8,7 @@ import useDeepCompareEffect from 'use-deep-compare-effect'
 import { ElementWithTooltip } from '../../../containers/widgets/ElementWithTooltip'
```

Comment:
This is the component where the bug surfaced, but no actual changes at this level (just variable rename + type updates)